app/models/user.rb

```diff
@@ -57,6 +59,14 @@ def email_activate!
     save!(validate: false)
   end
 
+  def enforce_password_validation
+    @enforce_password_validation = true
+  end
+
+  def password_required?
+    @enforce_password_validation || password.present?
+  end
```

Comment:
Why we need this method if it always return `true`?
Its better to use lambda `if: -> { password.present? }`.

Reply:
Variable `@enforce_password_validation` always be `nil`, because method `enforce_password_validation` is not used anywhere.
If you also want to validate empty password, you need add row `user.enforce_password_validation` to action update in users_controller.

Reply:
@evrusha Thanks! I used lambda and everything works.